specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-07-02-preview/examples/ListGuardrailsVersions.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "api-version": "2023-07-02-preview",
+    "subscriptionId": "subid1",
+    "location": "location1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/subid1/providers/Microsoft.ContainerService/locations/location1/guardrailsVersions",
```

Comment:
this should be /subscriptions/subid1/providers/Microsoft.ContainerService/locations/location1/guardrailsVersions/v1.0.0?

Reply:
but the ID of each should be shown. @weinong is correct here.

Reply:
Basically the list/get payloads should be identical except that the list has the `value` collection. Other than that though, there's no difference between a list and a get in terms of what they show. Get shows the full ID (including `v1.0.0`) and so does list

Reply:
Ohh I see. Ok, let me make the changes

Reply:
Yeah, makes sense. I was assuming that we were talking about list path. My bad. The response should have version. I made the changes. Pls take a look.